### ISSUE TITLE
Sleaker Sliders

### ISSUE THREAD

**Issue by bryevdv** [bokeh/bokeh#10081]
Our sliders seem *really* overly large and clunky to me. I feel like even a minor tweek could make them substantially more attractive. Below is before/after with some small experimentation in the browser CSS editor:

<img width="322" alt="Screen Shot 2020-05-27 at 5 40 41 PM" src="https://user-images.githubusercontent.com/1078448/83085572-37efc400-a041-11ea-93a8-487c9d52f403.png">
<img width="312" alt="Screen Shot 2020-05-27 at 5 40 18 PM" src="https://user-images.githubusercontent.com/1078448/83085571-36260080-a041-11ea-9e1e-3efa74c1a6ec.png">

Thoughts @bokeh/dev ? @mattpap are there any special considerations to changes like this? AFAIK the CSS changes do not/should not affect the layout at all.

**Comment by pavithraes** [comment]
I like the slimmer sliders. :)

**Comment by mattpap** [comment]
> are there any special considerations to changes like this?

Not really. Layout will adapt to CSS changes. Just make sure you follow how handle dimensions and position are computed, because it's easy to break the slider.

**Comment by howdoyouworkthis** [comment]
Slimmer is good. I have some other suggestions to fall in line with modern trendy material design - making the radii consistent between the slider bar and the handle, using related colors (or even gradients), or dropping the drop shadows. You could even then the bars out to just a line. :)

**Comment by bryevdv** [comment]
I'm moving this out to `future` because I'd like to make a whole, concerted effort to overhaul the visual style of our built-in widgets as well as our plot defaults, but that's really 3.0 territory (but we should start trying to plan for it sooner, rather than later)
